packages/grid/x-grid/src/tests/selection.XGrid.test.tsx

```diff
@@ -138,6 +138,22 @@ describe('<XGrid /> - Selection', () => {
     expect(getSelectedRows(apiRef)).to.deep.equal([0]);
   });
 
+  it('should call onSelectionModelChange when selection state changes', () => {
+    const onSelectionModelChange = spy();
```

Comment:
A small detail, I believe that we have used this convention in the codebase.

```suggestion
    const handleSelectionModelChange = spy();
```